docs/insomnia/design-documents.md

```diff
@@ -5,40 +5,54 @@ category: "API Design"
+**Note**: If you don't see a Document or Collection you're looking for, ensure you're in the right [Project](/insomnia/projects) by clicking on the Insomnia dropdown menu.  
+
+![Click Dashboard in the upper left-hand corner to return to a listing of your Documents and Collections.](/assets/images/access-dashboard.png)
+_From inside a Document or Collection, return to your listings by clicking Dashboard._
+
+For information about working with Collections, see [Request Collections](/insomnia/request-collections).
 
-1. Go to the Dashboard
-2. Click "Create" button in upper right hand corner
-3. Select "New Design Document" or a method of importing a
-4. Select how you would like to create a new document.
+## Create a New Design Document
+
+1. Go to the Dashboard.
+2. Click the **Create** button in upper right hand corner.
+3. From the dropdown menu, select **Design Document** or a method of importing.
+    * If you selected **Design Document**, enter a Document name and click **Create**. Spaces and special characters are allowed in your Document name.
+    * If you selected a method of importing, a modal will open prompting you to import your Document. 
 
 {:.alert .alert-primary}
-**Note**: When naming your document, the name selected will be displayed on the Dashboard. Spaces and special characters are allowed. 🤗
+**Note**: Once created, the Document name will be displayed next to the Dashboard navigation link. 
 
 ## Rename a Design Document
 
-1. Go to the Dashboard
-2. Open the action menu (visualized by the three dots) on the document you wish to rename.
-3. Select "Rename" action
-4. Rename the document
+There are two ways to rename a Design Document - from the dashboard and from inside the Design Document. 
+
+### Rename From the Dashboard
+
+1. Open the action menu (visualized by the three dots) on the document you wish to rename.
+2. Select **Rename** action.
+3. Rename the document.
+
+### Rename From Inside the Design Document
+
+1. Click on the dropdown arrow next to the Document name. 
+2. Click **Document Settings**. 
+3. Rename the document. It will auto-save. 
 
 ## Delete a Design Document
 
-1. Go to the Dashboard
+1. Go to the Dashboard.
 2. Open the action menu (visualized by the three dots) on the document you wish to rename.
-3. Select the "Delete" action
-4. Follow the steps to delete the document
+3. Click **Delete**.
+4. Confirm you want to delete the Document. 
 
 ## Edit a Design Document
 
-1. Go to the Dashboard
-2. Click on the document you wish to edit
-3. You will now be presented with the Editor Mode, and a Preview Pane will display the rendered results of the document you are currently editing.
-
-## Go Back to the Dashboard
-
-Click on the "Dashboard" link in the upper left hand corner to get back to the Dashboard
+1. Go to the Dashboard.
+2. Click on the Document you want to edit.
+3. You'll see **Editor Mode**, and a Preview Pane will display the rendered results of the document you are currently editing.
```

Comment:
By default this will show the Design tab with the preview pane, but if you navigate around it will show the last view that was opened. Ie. if you go to the test tab, back to dashboard, and open the document again, you'll see the test tab.